joblib/test/test_memory.py

```diff
@@ -303,9 +300,8 @@ def name_collision(x):
 def test_memory_warning_lambda_collisions(tmpdir):
     # Check that multiple use of lambda will raise collisions
     memory = Memory(location=tmpdir.strpath, verbose=0)
-    a = lambda x: x
-    a = memory.cache(a)
-    b = lambda x: x + 1
+     a = memory.cache(lambda x: x)
+    b = lambda x: x + 1  # noqa: 731
     b = memory.cache(b)
```

Comment:
I would use the same

```suggestion
    b = memory.cache(lambda x: x + 1)
```